src/transformers/models/seamless_m4t/modeling_seamless_m4t.py

```diff
@@ -3715,6 +3722,7 @@ def __init__(self, config):
         self.lm_head = nn.Linear(config.hidden_size, config.vocab_size, bias=False)
 
         # Initialize weights and apply final processing
+        self.shared = self.lm_head
```

Comment:
I can't see this is used within `SeamlessM4TForSpeechToSpeech`

(unlike the above change in `SeamlessM4TForSpeechToText`)

Reply:
I'll remove this in favor of https://github.com/huggingface/transformers/pull/27240